src/wpt/harness.ts

```diff
@@ -66,8 +68,208 @@ type TestCase = {
   test(_: unknown, env: Env): Promise<void>;
 };
 
+type UnknownFunc = (...args: unknown[]) => unknown;
+
+enum TestPhase {
+  INITIAL = 0,
+  STARTED = 1,
+  HAS_RESULT = 2,
+  CLEANING = 3,
+  COMPLETE = 4,
+}
+
+/**
+ * @class
+ *
+ * A single subtest. A Test is not constructed directly but via the
+ * :js:func:`test`, :js:func:`async_test` or :js:func:`promise_test` functions.
+ *
+ * @param {string} name - This must be unique in a given file and must be
+ * invariant between runs.
+ *
+ */
```

Comment:
We can remove the unnecessary parts from jsdoc, since we are using typescript. (such as line 82)